lib/OpenQA/Controller/Admin/Table.pm

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2014 SUSE Linux Products GmbH
```

Comment:
Wrong year and company, isn't it? And btw. I prefer GPL header without FSF address, but URL instead, once they move again it'll create tens of rpmlint warnings. And is recommended by https://www.gnu.org/licenses/gpl-howto.en.html